### ISSUE TITLE
Missing padding between elements inside Form Settings

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1644]
The Form Settings modal lacks space between elements, making it hard for certain users to use it.
![image](https://github.com/user-attachments/assets/3e70d674-dce2-4a24-a675-c31af77d7d4b)
